api/envoy/service/auth/v3/external_auth.proto

```diff
@@ -27,6 +27,24 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // A generic interface for performing authorization check on incoming
 // requests to a networked service.
 service Authorization {
+  // For v2alpha to work as expected, we need to convince
+  // proto_sync.py that the v2alpha package is the previous version of
+  // either this package or the v2 package. And to do that, you'd
+  // "normally" use the
+  // "(udpa.annotations.versioning).previous_message_type" option; but
+  // we don't have a 'message' to put it on, we only have a 'service'.
+  // Fortunately, proto_sync.py's parser is sloppy, and we can fool it
+  // with a comment:
+  //
+  //   previous_message_type = "envoy.service.auth.v2alpha.Authorization";
```

Comment:
While this trick may work, I think we're really relying on undocumented behavior in proto_sync.py. I'd prefer to keep the existing special cases, since they will sunset when v2 support is removed, there's not much benefit from optimizing for v2 consistency.